packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -431,11 +424,6 @@ function registerCommands(
     forceFunctionDebugInvoke
   );
 
-  const forceFunctionStopCmd = vscode.commands.registerCommand(
-    'sfdx.force.function.stop',
-    forceFunctionStop
-  );
```

Comment:
This is what wires in the command palette to have the `SFDX: Stop Function` command which is still necessary I believe